cg/cli/upload/gens.py

```diff
@@ -23,7 +23,7 @@
 @ARGUMENT_CASE_ID
 @DRY_RUN
 @click.pass_obj
-def upload_to_gens(context: CGConfig, case_id: str | None, dry_run: bool):
+def upload_to_gens(context: CGConfig, case_id: str | None, dry_run: bool, re_upload : bool):
```

Comment:
If adding a  new parameter to a CLI command you need to add it with a decorator for it to be passed along. Something like this
```suggestion
@click.option("--re-upload", is_flag=True, help="<something descriptive>")
def upload_to_gens(context: CGConfig, case_id: str | None, dry_run: bool, re_upload : bool):
```

Reply:
Yeah, I just copied those names from the genotype & coverage modules. Also thought it wasn't great to have so many different names. I'll replace the ones I can with `replace`.